docs/source/io_formats/statepoint.rst

```diff
@@ -108,6 +108,8 @@ The current version of the statepoint file format is 17.0.
              interpolation. Only used for 'energyfunction' filters.
            - **y** (*double[]*) -- Interpolant values for energyfunction
              interpolation. Only used for 'energyfunction' filters.
+           - **interpolation** (*int*) -- Interpolation type. Only used for
```

Comment:
We typically store the interpolation as an attribute, not a separate dataset. It would probably make the most sense here to put that attribute on the `y` dataset.